streams/transform-streams/flush.js

```diff
@@ -20,17 +20,20 @@ promise_test(() => {
 }, 'TransformStream flush is called immediately when the writable is closed, if no writes are queued');
 
 // TODO
```

Comment:
Any idea what this TODO comment is doing here?

Reply:
I was trying to ignore it.

It turns out it was added by @marvinhagemeister a year ago when he ported the tests to wpt format in https://github.com/whatwg/streams/pull/531. I can find nothing in that test or the surrounding ones that needs doing, so I have removed it.

Marvin, if you remember what needed doing here, please speak up!

Reply:
Just went through my original PRI honestly can't remember why I put it there :/